sentry_link/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@ueman you're the author AFAIU.
Can you re-license this under MIT to align with the other sentry-dart packages?

Reply:
That's correct. Yep, no problem, I can do so.

Reply:
Out of curiosity, what difference does that make?

Reply:
I'm no expert on this topic. Just that if it makes no difference to the contributor of the code, it sounds simpler when it's just a single license.